group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -17848,6 +17921,104 @@ public void testStreamsRebalanceTimeoutExpiration() {
         context.assertNoRebalanceTimeout(groupId, memberId1);
     }
 
+    @Test
+    public void testStreamsOnNewMetadataImage() {
```

Comment:
I don't understand this test, ie, how is it related to test code changes above?

Reply:
Did miss the PR comment... Sorry. -- In general we should avoid such piggy-backing to not confuse reviewers (ie, me :) )